doc/articles/features/PasswordVault.md

```diff
@@ -12,10 +12,10 @@ uid: Uno.Features.PasswordVault
 
 ## Supported features
 
-| Feature              | Windows | Android | iOS     | Web (WASM) | Catalyst | AppKit  | Linux (Skia) | Win 7 (Skia) | Tizen |
-|----------------------|---------|---------|---------|------------|----------|---------|--------------|--------------|-------|
-| `PasswordVault`      | ✔       | ✔       | ✔       | ✖          | ✔        | ✖       | ✖            | ✖            | ✖     |
-| `PasswordCredential` | ✔       | Partial | Partial | Partial    | Partial  | Partial | ✖            | ✖            | ✖     |
+| Feature              | Windows | Android | iOS     | Web (WASM) | AppKit  | Linux (Skia) | Win 7 (Skia) | Tizen |
+|----------------------|---------|---------|---------|------------|---------|--------------|--------------|-------|
+| `PasswordVault`      | ✔       | ✔       | ✔       | ✖           | ✖       | ✖            | ✖            | ✖     |
+| `PasswordCredential` | ✔       | Partial | Partial | Partial    | Partial | ✖            | ✖            | ✖     |
```

Comment:
not strictly related to the PR stated purpose but it `AppKit` was not removed in the previous update, the Skia columns also seems out of date